tools/topology/topology1/sof-imx8ulp-btsco.m4

```diff
@@ -76,10 +76,21 @@ DAI_ADD(sof/pipe-dai-capture.m4,
 dnl PCM_DUPLEX_ADD(name, pcm_id, playback, capture)
 PCM_DUPLEX_ADD(Port0, 0, PIPELINE_PCM_1, PIPELINE_PCM_2)
 
+# BCLK frequency is computed using the following formula:
+#	Freq(BCLK) = Freq(SYNC) * TDM_SLOTS * TDM_SLOT_WIDTH
+#
+# For 8ULP this yields the following frequencies (based on
+# supported BT HFP configurations):
+#
+#	1) NBS (Freq(FSYNC) = 8k)
+#		Freq(BCLK) = 8k * 16 * 1 = 128000
+#
+#	2) WBS (Freq(FSYNC) = 16k)
+#		Freq(BCLK) = 16k * 16 * 1 = 256000
 dnl DAI_CONFIG(type, idx, link_id, name, sai_config)
 DAI_CONFIG(SAI, 5, 0, sai5-bt-sco-pcm-wb,
 	SAI_CONFIG(I2S, SAI_CLOCK(mclk, 12288000, codec_mclk_out),
-		SAI_CLOCK(bclk, 256000, codec_consumer),
+		SAI_CLOCK(bclk, `eval(RATE * 16)', codec_consumer),
```

Comment:
You could also make RATE less ambiguous and rename as FSYNC ?

Reply:
good idea, renamed to `FSYNC_RATE`